PresentationLayer/Sources/PresentationLayer/Features/NextToGo/UIComponents/Filters/FiltersView.swift

```diff
@@ -21,21 +23,17 @@ struct FiltersView: View {
         HStack {
 
             ForEach(viewModel.filters) { filter in
-
-                HStack(spacing: 10) {
-
+                HStack(spacing: 8) {
                     Image(filter.category.iconName, bundle: .module)
-                        .font(.largeTitle)
-                        .scaleEffect(1.4)
+                        .font(sizeCategory >= .accessibilityExtraLarge ? .title : .largeTitle)
```

Comment:
Yeahh, accessibility font scaling rules to certain elements are not fun. Apple HIG says, don't just scale-up everything. Scale only things that matter to the users. They also say certain SF symbols can be scaled up if used as system font style. ;) 

https://developer.apple.com/design/human-interface-guidelines/accessibility